src/cmd/dev_test.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/defenseunicorns/uds-cli/src/types"
+	"github.com/stretchr/testify/assert"
```

Comment:
Recommend `require` over `assert` because `require` will fail the test faster (unless you find value in continuing the test suite after a test has failed)